src/tests/tic-tac-toe.test.ts

```diff
@@ -0,0 +1,21 @@
+import Game from "../tic-tac-toe";
+
+describe("Tic-tac-toe", () => {
+  let game: Game;
+  beforeEach(() => {
+    game = new Game();
+  });
+  it("starts with an empty board", () => {
+    const board = [["", "", ""], ["", "", ""], ["", "", ""]];
+    expect(game.board).toEqual(board);
+  });
+  it("places an X on the board on X's turn", () => {
+    game.place(0, 0);
+    expect(game.board[0][0]).toEqual("X");
+  });
+  it("switches players every turn", () => {
```

Comment:
This is sorta only testing one player's turn: if both placements were "0" the test would still pass.

Reply:
I can add two expectations inside this test. One that expects the first move is 'X', and the other to expect that the second move is 'O'.